android/engine/src/main/java/org/smartregister/fhircore/engine/data/local/DefaultRepository.kt

```diff
@@ -113,10 +133,37 @@ constructor(open val fhirEngine: FhirEngine, open val dispatcherProvider: Dispat
 
   suspend fun getBinary(id: String): Binary = fhirEngine.get(id)
 
-  suspend fun save(resource: Resource) {
+  suspend fun create(vararg resource: Resource): List<String> {
+    val syncStrategy = mutableMapOf<String, List<String>>()
```

Comment:
Is the list a proper data structure to use? I can see `PRACTITIONER_DETAILS_USER_DETAIL` is a single object and needs to be wrapped into a list like the others.

Reply:
Honestly, I did that because it's easier to check the mandatory tags later.

Is that the right thing?

Reply:
Okay